qiskit/mapper/layout.py

```diff
@@ -42,14 +44,23 @@ def __repr__(self):
     def from_dict(self, input_dict):
         """
         Populates a Layout from a dictionary.
+        The dictionary must be a bijective mapping between
+        virtual qubits (tuple) and physical qubits (int).
 
         Args:
-            input_dict (dict): For example,
-            {(QuantumRegister(3, 'qr'), 0): 0,
-             (QuantumRegister(3, 'qr'), 1): 1,
-             (QuantumRegister(3, 'qr'), 2): 2}
+            input_dict (dict):
+
+                e.g.:
+                virtual to physical:
+                    {qr[0]: 0,
```

Comment:
I would prefer this way to documenting the input type `(QuantumRegister(3, 'qr'), 0): 0`, since it might not be obvious what `qr` is.

Reply:
ok i updated it to show both ways.